src/plugins/intel_cpu/src/node.cpp

```diff
@@ -545,6 +545,17 @@ std::vector<memory::format_tag> Node::getAvailableFormatsForDims(const Shape &di
     return {memory::format_tag::any};
 }
 
+static void fetchRawMemory(const MemoryPtr& mem) {
+    // TODO: conceptually fetchRawMemory is a very bad solution
+    if (mem->getDesc().getPrecision() == element::string) {
+        return;
+    }
+    auto block = mem->getMemoryBlock();
+    if (mem->isDefined()) {
+        block->resize(mem->getSize());
+    }
```

Comment:
Possible solutions:
1. `allocate()` for the memory interface. The main drawback is that the memory with such an interface don't have to allocate memory in constructor, which brings additional complexity: `allocated()` check and subsequent mandatory `allocate()` call, which obviously is extra work for the user.
2. lazy evaluation inside `getData()`. There are two problems, the first one is that in this case the `getData` call wouldn't be constant anymore thereby wouldn't be thread safe. Another problem is that some implementations still use dnnl memory object instead of the plugin entities, thus they wouldn't call `getData` and wouldn't trigger memory reallocation.
Probably we need another solution.